packages/core/jkl/_motion.scss

```diff
@@ -27,6 +27,7 @@ $_timings: (
     "energetic": 75ms,
     "snappy": 100ms,
     "productive": 150ms,
+    "polite": 200ms,
```

Comment:
Naming things is hard. Åpen for andre forslag her. Kanskje ikke sååå farlig for et "L1-navn"?